addons/mail/static/tests/new/discuss/discuss_tests.js

```diff
@@ -129,6 +126,10 @@ QUnit.module("mail", (hooks) => {
     });
 
     QUnit.test("can create a new channel", async (assert) => {
+        // for autocomplete stuff
+        patchWithCleanup(browser, {
+            setTimeout: (fn) => fn(),
+        });
```

Comment:
I've moved the `patchWithCleanup()` of `Browser/setTimeout`  to the specific tests that use `channel_selector`, as apparently `Autocomplete` component still needs some help from this patching in test.